### PR TITLE
Add support for GroupSequence

### PR DESCRIPTION
**Pull request by emilien-puget**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #2009 
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Comment by soyuka** [comment]
Neat, I see this as a new feature though.

**Comment by emilien-puget** [comment]
I am not sure, there is an entry in the documentation (https://api-platform.com/docs/core/validation#using-validation-groups) that mention them

File: src/Bridge/Symfony/Validator/Validator.php [hunk 1]
```diff
@@ -52,7 +53,9 @@ public function validate($data, array $context = [])
                 $validationGroups = $validationGroups($data);
             }
 
-            $validationGroups = (array) $validationGroups;
+            if (!($validationGroups instanceof GroupSequence)) {
```
Comment (dunglas) [inline src/Bridge/Symfony/Validator/Validator.php]: Inner parenthesis can be dropped.

**Comment by dunglas** [comment]
Thank you very much @emilien-puget

**Comment by emilien-puget** [comment]
It looks like the problem is also in the master branch, do you want me to do another PR ?

**Comment by dunglas** [comment]
No it's ok @emilien-puget, we merge lower branches in master regularly. Your commit we'll be in master in a few hours or days.

**Comment by emilien-puget** [comment]
Ok, thank you. I will close the issue then.

**Pull request merged**